prepublish.js

```diff
@@ -14,6 +14,7 @@ const {
 	"@types/react": typesReact,
 	"@types/react-dom": typesReactDom,
 	"web-vitals": webVitals,
+	"@rollup/plugin-typescript": rollupTypescriptPlugin,
```

Comment:
It's weird that `@rollup/plugin-typescript` is in the dependencies in the first place. It should be in the developer dependencies instead, a more permanent fix would be to move it there. I'll potentially make a new PR for updating the dependencies in the recent future - if not, you can make the change and update this PR!

Thanks for your contribution :)

Reply:
Yeah, I found it weird that `devDependencies` were just not used at all. Thanks for fixing 🙏🏼